prebid-server/pbs-modules/greenbids-analytics-reporter.md

```diff
@@ -32,24 +32,73 @@ Here's a description of the global parameters:
 | exploratory-sampling-split | float | Exploration vs Exploitation ratio of analytics traffic |
 | timeout-ms | int | Timeout limit on sending POST request to Greenbids Analytics Server |
 
-The Greenbids Analytics module also includes account-specific parameters for each publisher setup. These parameters should be included in the extension of the bid request json: `ext.prebid.analytics.greenbids`
+The Greenbids Analytics module includes account-specific parameters for the publisher setup. The logic of the config is as follows:
 
-| Parameter | Scope | Description | Example | Type |
-| --------- | ---- | ------------- | ------------- | ----------- |
-| pbuid | required | The Greenbids Publisher ID | greenbids-publisher-1 | string |
-| greenbidsSampling | optional  | sampling factor [0-1] (a value of 0.1 will filter 90% of the traffic) | 1.0  | float |
+- `BidRequest` extension in `ext.prebid.analytics.greenbids` if defined takes precedence over account configs.
+- If the `BidRequest` extension is not defined, the account config is used and defined one per individual publisher.
+  The config is stored in `yaml` file under path `settings.filesystem.settings-filename` of the Prebid config.
+- If the account config is not defined, the default account config is used. It is defined in `settings.default-account-config` field of the Prebid config.
```

Comment:
Not clear to me that default account config makes sense because it requires a specific greenbids account ID.

Reply:
We have single publisher activation and a pbuid at account level. In case if account config is not defined and we fallback to default account config the pbuid field can be left blank (because it is optional) which will just result in no filtering as no model on the null pbuid is fetched. I've added pbuid in case if we could potentially need default filtering behavior for all accounts.